package.json

```diff
@@ -1,9 +1,10 @@
 {
     "name": "@moovio/node",
-    "version": "1.0.2",
+    "version": "1.0.3",
     "description": "Node SDK for the Moov API and Dashboard",
     "module": "dist/mjs/index.js",
     "main": "dist/cjs/index.js",
+    "type": "module",
```

Comment:
Not sure we can do this with the dual ES/CJS build we have right now. @eric-moov can answer that, though.

Reply:
Ah that'd be great to know! I couldn't get tests to pass without this.